docs/getting-started-guides/mesos.md

```diff
@@ -104,7 +104,7 @@ $ km controller-manager \
 
 $ km scheduler \
   --address=${servicehost} \
-  --mesos-master=${MESOS_MASTER} \
+  --mesos-master=${servicehost}:5050 \
```

Comment:
nope. we should keep `${MESOS_MASTER}` and just change the docs above, removing the `http://` from it